website/content/api-docs/variables/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+layout: api
+page_title: Variables - HTTP API
+description: |-
+  The /var endpoints are used to query for and interact with variables and variable
+  locking.
+---
+
+# Vars HTTP API
+
+The `/var` and `/vars` endpoints are used to query for and interact with
+variables, and set up locks and leases over them.
+
+See the [Variables] documentation for information how these capabilities are
+used. For a CLI to perform these operations manually, please see the
+documentation for the [`nomad var`] commands.
```

Comment:
sorry I'm late to the review party, but it doesn't look like these links go anywhere.

I think ya need these down at the bottom of the file

```
[Variables]: /nomad/docs/concepts/variables
[`nomad var`]: /nomad/docs/commands/var
```

like you have in `locks.mdx`

Reply:
fixed!